README.md

```diff
@@ -27,9 +27,11 @@ They don't need to be perfect, complete, or even very good when you submit them.
 1. Fork this repository and create a new branch for your new RFC.
 2. Copy `template.md` into the `rfcs` folder and rename it to `my_feature.md`, where `my_feature` is a unique identifier for your feature.
 3. Fill out the RFC template with your vision for `my_feature`.
-4. [Create a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) in this repo, with a one-sentence description of what the RFC is about.
+4. [Create a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) in this repo. The first comment should include:
+   1. A one-sentence description of what the RFC is about.
+   2. A link to the "rendered" form of the `my_feature.md` file. To do so, link directly to the file on your own branch, so then the link stays up to date as the file's contents changes. See #1 for an example of what this looks like.
```

Comment:
`#1` is ambiguous here. Maybe used bulleted items instead?

Reply:
Markdown automatically converts nested numbered lists into i. ii. etc notation :) See: https://github.com/bevyengine/rfcs/blob/c3175d40ced82e764dec6a5190d7e6787072d8d2/README.md

Reply:
Cool cool. Carry on!